carbon-design-system/carbon-addons-iot-react
# (feat): Consistent  header for CardEditor

Closes #1561

**Summary**

- This PR uses a consistently styled header that is part of the `CardEditor` component for both the "Gallery" section and the "Add a card" section. We no longer utilize `SimpleList` header

**Change List (commits, features, bugs, etc)**

- Added the "gallery" header section to `CardEditor.jsx`
- change `title` prop for `SimpleList` in `CardGalleryList.jsx`
- add styles to `_card-editor.scss`

**Acceptance Test (how to verify the PR)**

- Go to Dashboard editor [story](https://deploy-preview-1870--carbon-addons-iot-react.netlify.app/)
- Click on card type to add. 
- Click on "Add a card" header
- see that styles look the same as production with the change of making the "add to card" header have the same height as the "Gallery" header.
